graphql/mutation_resolver.go

```diff
@@ -852,7 +852,15 @@ func (r *mutationResolver) RemoveVolume(ctx context.Context, volumeID string) (b
 }
 
 // UpdateSpawnHostStatus is the resolver for the updateSpawnHostStatus field.
-func (r *mutationResolver) UpdateSpawnHostStatus(ctx context.Context, hostID string, action SpawnHostStatusActions) (*restModel.APIHost, error) {
+func (r *mutationResolver) UpdateSpawnHostStatus(ctx context.Context, hostID string, action SpawnHostStatusActions, updateSpawnHostStatusInput *UpdateSpawnHostStatusInput) (*restModel.APIHost, error) {
+	shouldKeepOff := false
+	// TODO: Use input object throughout resolver once deprecated fields are removed
```

Comment:
Is this still relevant? Or is there a ticket to address this?

Reply:
I deprecated the fields in this PR, here's a followup ticket: DEVPROD-7046